src/benchmarklib/abstract_table_generator.cpp

```diff
@@ -404,6 +403,8 @@ void AbstractTableGenerator::_create_table_indexes(
 }
 
 AbstractTableGenerator::IndexesByTable AbstractTableGenerator::_indexes_by_table() const {
+  // Currently, indexes are not used by the benchmarks. Therefore it doesn't make sense to create them since it would
+  // just create unnecessary memory consumption.
```

Comment:
If someone adds indexes to concrete table generator classes, this comment is not valid anymore, and one might forget to change this comment. I would change this comment so that it's also valid if a concrete table generation implementation specifies indexes.

Suggestion:
```suggestion
  // Indexes can be specified in a derived concrete class by overriding this function.
```